Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/CancelRide.hs

```diff
@@ -110,5 +129,30 @@ cancelRideHandler ServiceHandle {..} requestorId rideId req = withLogTag ("rideI
             source = source,
             reasonCode = Just reasonCode,
             driverId = mbDriverId,
+            driverLat = lat,
+            driverLon = lon,
+            driverDistToPickup = disToPickup,
             ..
           }
+
+getDistanceBetweenPoints ::
+  ( EncFlow m r,
+    CacheFlow m r,
+    EsqDBFlow m r,
+    Metrics.CoreMetrics m,
+    Maps.HasCoordinates a,
+    Maps.HasCoordinates b
+  ) =>
+  Id Merchant ->
+  a ->
+  b ->
+  m Meters
+getDistanceBetweenPoints merchantId a b = do
```

Comment:
Don't use `a` and `b` as variable name.
Rename `getDistanceBetweenPoints` properly.